<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/images/SecurityLock.png</normaloff>:/images/SecurityLock.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Company Information</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Company Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="5">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Postcode:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Town:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Telephone:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Telephone 2:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>325</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Users</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="SaveAndNew">
            <property name="text">
             <string>Save And New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SaveChanges">
            <property name="text">
             <string>Save Changes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Delete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>User Details</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Login Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="LoginName"/>
           </item>
           <item row="0" column="7" colspan="6">
            <widget class="QLineEdit" name="FullName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="EditHR">
             <property name="text">
              <string>Can Edit HR Data</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QCheckBox" name="EditFinance">
             <property name="text">
              <string>Can Edit Finance Data</string>
             </property>
            </widget>
           </item>
           <item row="1" column="6" colspan="2">
            <widget class="QCheckBox" name="EditLeaves">
             <property name="text">
              <string>Can Edit Leaves</string>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QCheckBox" name="EditPayroll">
             <property name="text">
              <string>Can view payroll</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLineEdit" name="Password">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QCheckBox" name="EditAttendance">
             <property name="text">
              <string>Can Edit Attendance</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Full Name:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>SaveAndNew</tabstop>
  <tabstop>SaveChanges</tabstop>
  <tabstop>Delete</tabstop>
  <tabstop>LoginName</tabstop>
  <tabstop>Password</tabstop>
  <tabstop>FullName</tabstop>
  <tabstop>EditHR</tabstop>
  <tabstop>EditFinance</tabstop>
  <tabstop>EditAttendance</tabstop>
  <tabstop>EditLeaves</tabstop>
  <tabstop>EditPayroll</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_3</tabstop>
 </tabstops>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
